src/components/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+function Nav() {
+  return (
+    <header className="Navbar">
+      <nav className="Nav-header">
+        <h1 className="Nav-logo">
+          <NavLink to="/bookstore/" rel="noopener noreferrer">
+            Bookstore CMS
+          </NavLink>
+        </h1>
+        <ul>
+          <li>
+            <NavLink to="/bookstore/" rel="noopener noreferrer">
```

Comment:
- [ ] Kindly change this **`to`** attribute here to match the root path defined in the App component.

Reply:
Change made